NextcloudTalk/NewRoomTableViewController.swift

```diff
@@ -0,0 +1,303 @@
+//
+// Copyright (c) 2024 Ivan Sein <ivan@nextcloud.com>
+//
+// Author Ivan Sein <ivan@nextcloud.com>
+//
+// GNU GPL version 3 or any later version
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import UIKit
+
+enum NewRoomOption: Int {
+    case createNewRoom = 0
+    case joinOpenRoom
+}
+
+@objcMembers class NewRoomTableViewController: UITableViewController,
+                                               UINavigationControllerDelegate,
+                                               UISearchResultsUpdating {
+
+    var account: TalkAccount
+
+    var indexes: [String] = [""]
+    var contacts: [String: [NCUser]] = [:]
+
+    var searchController: UISearchController
+    let resultTableViewController = ContactsSearchResultTableViewController(style: .insetGrouped)
+
+    var searchTimer: Timer?
+    var searchRequest: URLSessionTask?
+
+    init(account: TalkAccount) {
+        self.account = account
+        self.searchController = UISearchController(searchResultsController: resultTableViewController)
+        super.init(style: .insetGrouped)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        self.navigationItem.hidesSearchBarWhenScrolling = false
+        self.getPossibleContacts()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor: NCAppBranding.themeTextColor()]
```

Comment:
Could use https://github.com/nextcloud/talk-ios/blob/master/NextcloudTalk/NCAppBranding.m#L228 until we have a super class for this.

Reply:
Forgot about that function. I agree, we should use it until we create a super class 👍